lib/entity_cache/entity_cache.rb

```diff
@@ -20,10 +20,9 @@ def self.build(subject, persistent_store: nil, write_behind_delay: nil)
 
     Clock::UTC.configure instance
     Telemetry::Logger.configure instance
-
     Storage::Temporary.configure instance, subject
 
-    Storage::Persistent.configure instance, subject, implementation: persistent_store
```

Comment:
Although, that was good wording for supplying an implementation. Gotta remember that for similar patterns.